### PR TITLE
Demo: Same templates for components and helpers, also icons

### PR DESCRIPTION
**Pull request by literat**
## Description

* icons in both packages use the same partial
* helpers and components in both packages use the same partial

### Issue reference

https://jira.almacareer.tech/browse/DS-1231

### CODE REVIEW

File: apps/demo/partials/web-react/index.hbs [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-    {{#each @web_react_components as |component|}}
-      {{> component component=component path="src/components"}}
-    {{/each}}
-  {{/components }}
+  {{#> section title="Components"}}
+    {{#> boxList }}
+      {{#each @web_react_components as |component|}}
+        {{> component component=component path="src/components"}}
+      {{/each}}
+    {{/boxList }}
```
Comment (crishpeen) [inline apps/demo/partials/web-react/index.hbs:8]:
```suggestion
    {{/boxList}}
```

File: apps/demo/partials/web/helpers.hbs [hunk 2]
```diff
@@ -1,18 +1,17 @@
 {{#> web/layout/default page="helpers" }}
 
-  <section class="docs-Section">
+  {{#> section title="Helpers"}}
 
-    <h2 class="typography-heading-medium-text text-secondary-disabled mb-700 mb-tablet-900">Helpers</h2>
-    <ul class="Grid Grid--cols-2 Grid--tablet--cols-3 list-unstyled">
+    {{#> boxList}}
```
Comment (crishpeen) [inline apps/demo/partials/web/helpers.hbs:5]:
```suggestion
    {{#> boxList }}
```

Nitpicking, but we are not consistent in how we write handlebars stuff.

File: apps/demo/partials/web/index.hbs [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
-    {{#each @web_components as |component|}}
-      {{> component component=component path="src/scss/components"}}
-    {{/each}}
-  {{/components }}
+  {{#>section title="Components"}}
+    {{#> boxList }}
+      {{#each @web_components as |component|}}
+        {{> component component=component path="src/scss/components"}}
+      {{/each}}
+    {{/boxList }}
```
Comment (crishpeen) [inline apps/demo/partials/web/index.hbs:8]:
```suggestion
    {{/boxList}}
```

**Review by crishpeen — APPROVED** [review]
does it match formating of our Twig demo? If not, we should edit it there as well.

**Review by pavelklibani — APPROVED** [review]
good idea 👍 thank you

File: apps/demo/partials/web-react/index.hbs (hunk 1 above)
Comment (literat) [reply apps/demo/partials/web-react/index.hbs]: Fixed.

File: apps/demo/partials/web/index.hbs (hunk 3 above)
Comment (literat) [reply apps/demo/partials/web/index.hbs]: Fixed.

**Pull request merged**